ods_ci/libs/DataSciencePipelinesAPI.py

```diff
@@ -48,7 +48,9 @@ def wait_until_openshift_pipelines_operator_is_deployed(self):
         )
 
     @keyword
-    def login_using_user_and_password(self, user, pwd, project):
+    def login_and_wait_dsp_route(
```

Comment:
ds-pipeline-ui-sample in RHODS won't be deployed unless you use custom DataSciencePipelineApplication.yaml

I think the default route where the API is served has this name: 
ds-pipeline-pipelines-definition

Reply:
Done